CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 # Upcoming release
 
 ### Added
+- **FileManager**
+- Added `decode(from:as:using:)'` method to decode Decodable object previously saved to a file. [#599](https://github.com/SwifterSwift/SwifterSwift/pull/599) by [AlexeiGitH](https://github.com/AlexeiGitH)
```

Comment:
Here `to decode Decodable object` could be `to decode a Decodable object`? English is not my native language so this is more a question 😆